docs/en/latest/plugins/prometheus.md

```diff
@@ -120,12 +120,56 @@ Or you can goto [Grafana official](https://grafana.com/grafana/dashboards/11719)
 
 ### Available metrics
 
-* `Status codes`: HTTP status codes returned by upstream services. These are available per service and across all services.
+* `Status codes`: HTTP status code returned from upstream services. These status code available per service and across all services.
+
+    Attributes：
+
+    | Name         | Description         |
+    | -------------| --------------------|
+    | code         | The HTTP status code returned by the upstream service. |
+    | route        | The `route_id` of the matched route is request. If it does not match, the default value is an empty string. |
+    | matched_uri  | The `uri` of the route matching the request, if it does not match, the default value is an empty string. |
+    | matched_host | The `host` of the route that matches the request. If it does not match, the default value is an empty string. |
+    | service      | The `service_id` of the route matched by the request. When the route lacks service_id, the default is `$host`. |
+    | consumer     | The `consumer_name` of the consumer that matches the request. If it does not match, the default value is an empty string. |
+    | node         | The `ip` of the upstream node. |
+
 * `Bandwidth`: Total Bandwidth (egress/ingress) flowing through apisix. This metric is available per service and as a sum across all services.
+
+    Attributes：
+
+    | Name         | Description |
+    | -------------| ------------- |
+    | type         | The type of bandwidth(egress/ingress). |
+    | route        | The `route_id` of the matched route is request. If it does not match, the default value is an empty string.. |
+    | service      | The `service_id` of the route matched by the request. When the route lacks service_id, the default is `$host`. |
+    | consumer     | The `consumer_name` of the consumer that matches the request. If it does not match, the default value is an empty string. |
+    | node         | The `ip` of the upstream node. |
+
 * `etcd reachability`: A gauge type with a value of 0 or 1, representing if etcd can be reached by a apisix or not.
 * `Connections`: Various Nginx connection metrics like active, reading, writing, and number of accepted connections.
 * `Batch process entries`: A gauge type, when we use plugins and the plugin used batch process to send data, such as: sys logger, http logger, sls logger, tcp logger, udp logger and zipkin, then the entries which hasn't been sent in batch process will be counted in the metrics.
 * `Latency`: The per service histogram of request time and the overhead added by APISIX (request time - upstream response time).
+
+    Attributes：
+
+    | Name      | Description |
+    | ----------| ------------- |
+    | type      | The type of request. |
```

Comment:
What is the type of request? It's hard to understand.

Reply:
updated.